modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -184,7 +185,7 @@ private void _writeBody(JspWriter jspWriter) throws Exception {
 		jspWriter.write("</div><div class=\"autofit-col autofit-col-expand");
 		jspWriter.write(" autofit-col-gutters\"><p class=\"card-title\"");
 
-		String title = getTitle();
+		String title = HtmlUtil.escape(getTitle());
```

Comment:
Do not escape it here, this is used as an attribute in `title="..."`, so rather do:

```
jspWriter.write(HtmlUtil.escapeAttribute(title));
```

This leaves `title` unescaped so it can be used in other contexts.

Below, we're apparently passing it to `LinkTag` so you probably want to actually do the escaping of `label` inside that component.